Week1/homework/9-log-the-remainder.js

```diff
@@ -0,0 +1,10 @@
+let x = 7;
+x = x % 3 ; // new value of x will be 1 as it's remainder when it's divided by three is 1.
+console.log(x); // You'll see it is 1.
+
+let y = 21;  // Their value would be the same. 
+let z = 13;
+
+console.log(y);  /* We can see they still have their initial values.
+ Because the remainder includes x but not y and z.*/
```

Comment:
I think there is some confusion regarding this question. What's required from this assignment is to print what you think the answer is for following statements:

```
1. If x equals 7, and the only other statement is x = x % 3, what would be the value of x after the calculation?
2. If y equals 21, and the only other statement is x = x % 4, what would be the value of y after the calculation?
3. If z equals 13, and the only other statement is x = x % 2, what would be the value of z after the calculation?
```

🙂

Reply:
My mistake. I think I didn't read the question correctly. Please ignore my comment 🙂